tests/ops/op_math/test_controlled_op.py

```diff
@@ -226,6 +226,75 @@ def test_no_matrix_if_batching(self):
         op = Controlled(base, 1)
         assert op.has_matrix is False
 
+    @pytest.mark.parametrize("cwires, cvalues", [(0, [0]), ([3, 0, 2], [1, 1, 0])])
+    def test_has_decomposition_true_via_control_values(self, cwires, cvalues):
+        """Test that Controlled claims `has_decomposition` to be true if there are
+        any negated control values."""
+
+        op = Controlled(qml.RX(0.2, wires=1), cwires, cvalues)
```

Comment:
Maybe `RX` isn't the best target gate for this, since it has a controlled version of itself. Creating a `DummyOp` is probably better for this situation.